### ISSUE TITLE
creating a database schema for CivicTechJobs

### ISSUE THREAD

**Issue by blulady** [hackforla/CivicTechJobs#417]
### Overview
Before building a database we should probably have a schema.

### Action Items
- [ ] create a database schema
- [ ] create an ERD in lucid chart (used in People Depot team)
- [ ] submit the database schema before the dev team
- [ ] confirm with the rest of the team that we correctly understand the work flow and needs of the team

### Resources/Instructions
https://www.databasestar.com/how-to-design-a-database/
https://www.databasestar.com/database-timezones/
https://www.lucidchart.com/blog/database-design-best-practices
https://www.conceptatech.com/blog/best-practices-how-to-design-a-database
[People Depot ERD](https://lucid.app/lucidchart/ac2f3e81-00d2-4257-b1fc-266d7f0a4cbe/view)

From the dev team discussion 23-07-11:
Job Postings:
    Job Posting name/id
    many to one relationship with project
    many to one relationship with users profile
    
UsersProfile/Project Managers:
    name
    many to one relationship with project
    one to many relationship with job postings
    
Projects:
    project name/id
    one to many relationship with users profile?
    one to many relationwhip with roles

**Comment by kcoronel** [comment]
@MattPereira @bzzz-coding since Sarah has off-boarded from the project I removed her from this issue and moved this back into the Icebox. Something I would add to the top of the issue to make it more specific would be whether this schema is for the CMS or for CTJ. Additionally, if for CMS (my assumption), we discussed not having a 1 to 1 relationship between project managers and CMS but instead having project emails/profiles have a 1 to 1 relationship with the CMS.
